src/fetchers/tarball-fetcher.js

```diff
@@ -93,12 +93,39 @@ export default class TarballFetcher extends BaseFetcher {
   } {
     const integrityInfo = this._supportedIntegrity();
 
+    const now = new Date();
```

Comment:
Move this closer to where it is used?

Reply:
No, it's here because I don't want each file of a single package to have a different timestamp (like it would be the case if the `Date` constructor was called inside the closure where it is used).